developer_docs/ansible_dev_setup.asciidoc

```diff
@@ -0,0 +1,138 @@
+= Ansible devel setup
+:toc: right
+:toclevels: 5
+
+[[prerequisites]]
+== Prerequisites
+
+* https://github.com/theforeman/foreman/blob/develop/developer_docs/foreman_dev_setup.asciidoc[Foreman devel setup]
+* https://github.com/theforeman/foreman/blob/develop/developer_docs/smart_proxy_dev_setup.asciidoc[Smart Proxy devel setup]
+* https://github.com/theforeman/foreman/blob/develop/developer_docs/remote_execution_dev_setup.asciidoc[Remote Execution devel setup]
+* Installed https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible]
+* Running machine that can be connected to via SSH (key authentication is best)
+
+[[foreman-plugin]]
+== Foreman plugin
+
+Download https://github.com/theforeman/foreman_ansible[foreman_ansible] plugin
+[source, bash]
+....
+git clone https://github.com/theforeman/foreman_ansible.git
+....
+
+Enable `foreman-ansible` plugin
+[source, ruby]
+....
+# bundler.d/ansible.local.rb
+
+gem 'foreman_ansible', path: '/path/to/foreman_ansible'
+....
+
+Install plugin dependencies & migrate database
+[source, bash]
+....
+bundle install
+npm install
+
+bundle exec rails db:migrate
+....
+
+
+[[smart-proxy-plugin]]
+== Smart Proxy plugin
+Add plugin gems
+
+[source, bash]
+....
+# /path/to/smart-proxy/bundler.d/ansible.local.rb
+
+gem 'smart_proxy_ansible', git: 'https://github.com/theforeman/smart_proxy_ansible.git'
+....
+
+Install gems
+[source, bash]
+....
+bundle install
+....
+
+Enable feature
+
+[source, yaml]
+....
+# config/settings.d/ansible.yml
+---
+:enabled: true
+....
+
+Run all servicies
+[source, bash]
+....
+bundle exec foreman start
+bundle exec rake dynflow:executor
```

Comment:
Guess you could do that, but if you go down that route, it would be better to also have a custom initializer with
```
# config/initializers/foreman_tasks.rb
if defined?(ForemanTasks) && Rails.env != 'test'
  ForemanTasks.dynflow.config.remote = true
end
```